nixos/modules/system/boot/systemd.nix

```diff
@@ -133,20 +133,27 @@ let
 
       # Slices / containers.
       "slices.target"
+    ] ++ (optionals systemd.withImportd [
+      "systemd-importd.service"
+    ]) ++ (optionals systemd.withMachined [
```

Comment:
```suggestion
    ] ++ optionals systemd.withImportd [
      "systemd-importd.service"
    ] ++ optionals systemd.withMachined [
```

Reply:
Thanks, but I'll leave those as-is for now as they're consistent with the existing `withCryptsetup` check (which you did not suggest changing) as well as the other `(optional[s] ...)` snippets.